web/index.html

```diff
@@ -13,108 +13,74 @@
 
     <title>Cigar2</title>
 
-    <link id="favicon" rel="icon" type="image/png" href="assets/img/favicon.png" />
+    <link id="favicon" rel="icon" type="image/png" href="assets/img/favicon.png">
     <link href="https://fonts.googleapis.com/css?family=Ubuntu:700" rel="stylesheet" type="text/css">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" rel="stylesheet">
     <link href="assets/css/index.css" rel="stylesheet">
+    <link href="assets/css/gallery.css" rel="stylesheet">
 
-    <script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
     <script src="./assets/js/Sound.js"></script>
     <script src="assets/js/main_out.js"></script>
 </head>
 
 <body>
-    <div class="modal fade" id="inPageModal" role="dialog">
```

Comment:
Did you add a replacement for this one

Reply:
Yes, the replacement element is #gallery